electrochemicalquantities.ttl

```diff
@@ -146,6 +153,14 @@ skos:prefLabel rdf:type owl:AnnotationProperty .
  ] .
 
 
+###  http://emmo.info/electrochemistry#electrochemistry_08833ed2_6324_411a_b34b_fe64c44cd5ef
+:electrochemistry_08833ed2_6324_411a_b34b_fe64c44cd5ef rdf:type owl:Class ;
```

Comment:
Is it necessary to further specify a `DischargeCurrentCapacity` when `DischargeCurrent `already exists?

Reply:
That quantity was added because it is included in the results files of some battery cyclers. I might change the prefLabel to ConstantCurrentDischargeCapacity to make it more clear, though.